backend/src/Services/Logging/LoggerImpl.ts

```diff
@@ -8,33 +8,42 @@ export class LoggerImpl {
     }
 
     debug(context?:ILoggingContext,  message?: any, ...optionalParams: any[]):void{
-        this.log(context, "debug", message, ...optionalParams);
+        this.log(context, "", message, ...optionalParams);
     }
 
     info(context?:ILoggingContext,  message?: any, ...optionalParams: any[]):void{
-        this.log(context, "info", message, ...optionalParams);
+        this.log(context, "", message, ...optionalParams);
     }
 
     warn(context?:ILoggingContext,  message?: any, ...optionalParams: any[]):void{
-        this.log(context, "WARN", message, ...optionalParams);
+        this.log(context, "WARN ", message, ...optionalParams);
```

Comment:
nitpick: why the added space?

Reply:
makes everything line up in monospace fonts :)